custom/ilsgateway/slab/handlers/transfer.py

```diff
@@ -0,0 +1,39 @@
+import re
+
+from datetime import datetime
+
+from custom.ilsgateway.models import SupplyPointStatus, SupplyPointStatusTypes, SupplyPointStatusValues
+from custom.ilsgateway.slab.messages import TRANS_HELP
+from custom.ilsgateway.tanzania.handlers.keyword import KeywordHandler
+from custom.ilsgateway.tanzania.reminders import SOH_CONFIRM
+
+
+class TransferHandler(KeywordHandler):
+
+    def help(self):
+        self.respond(TRANS_HELP)
```

Comment:
It's translated in respond: https://github.com/dimagi/commcare-hq/blob/master/custom/ilsgateway/tanzania/handlers/keyword.py#L58